UPGRADE.md

```diff
@@ -96,6 +96,9 @@ You can bring back previous configuration by overriding current routing with you
 
 * `AttributeValue::$localeCode` property was added to make it translatable. Now, every attribute value has a locale code to be displayed properly in different locales. All attribute values are migrated to the new concept with migration `Version20170109143010`. Look at [this PR](https://github.com/Sylius/Sylius/pull/7219) if you have any problems with upgrade.
 
+* `Sylius\Component\Attribute\Repository\AttributeRepositoryInterface` and its implementations were removed due to not being
+  used anymore. You can bring back missing `findByName` method by copying it from Git history to your codebase.
```

Comment:
What about referencing the last versions of the relevant files? https://github.com/Sylius/Sylius/blob/2ad4590e53126433671cf641d49d9e03ff43c5e0/src/Sylius/Bundle/AttributeBundle/Doctrine/ORM/AttributeRepository.php and https://github.com/Sylius/Sylius/blob/2ad4590e53126433671cf641d49d9e03ff43c5e0/src/Sylius/Component/Attribute/Repository/AttributeRepositoryInterface.php